python/py-reflink/Portfile

```diff
@@ -25,6 +25,10 @@ checksums           rmd160  59ff15dd3f67016c3b509f9c24517dda60595c5b \
 python.versions     37 38 39
 
 if {${name} ne ${subport}} {
+    # Backport https://gitlab.com/rubdos/pyreflink/-/merge_requests/10
+    patch.pre_args  -p1
+    patchfiles      before-sierra.patch
```

Comment:
Is this patch already present in the source tree?

Reply:
@mascguy I’ve just created it.

I’m backporting all patches which is created for MacPorts

Reply:
@mascguy well, now I understand your point. I just forgot to add patch file to commit :(

Reply:
No worries Kirill!